test/redux_tests.js

```diff
@@ -2,35 +2,44 @@ import assert from 'assert'
 import { createExperiments } from '../module/reducer'
 import { setExperimentVariant, loadExperimentVariants } from '../module/actions'
 
-const reducer = createExperiments({})
+const experiments = {
```

Comment:
I don't see any meaningful change in this file

Reply:
I changed the test for load and set.And now I added a new test for test